pkg/util/kubernetes/apiserver/types.go

```diff
@@ -38,4 +38,6 @@ const (
 	ReplicaSetsInformer InformerName = "replicaSets"
 	// ServicesInformer holds the name of the informer
 	ServicesInformer InformerName = "services"
+	// ServicesInformer holds the name of the informer
+	NodesInformer InformerName = "nodes"
```

Comment:
You probably mean this part, right? 
https://github.com/DataDog/datadog-agent/blob/8e8ed03f3053a15cb591b3e232911240a4022654/pkg/util/kubernetes/apiserver/metadata_controller.go#L49

I don't think so, we use the same informer cache and they are shared. On the above controller it waits for the cache to sync, our code does the same a little later. So we should be fine. 
At least from what I can tell. 

One could argue to refactor to use the same structure, but I am not 100% sure there. 

🤔

Reply:
After discussing with @DataDog/container-integrations we might want to do it in a follow up PR. Will close it for now. Feel free to unresolve in case I am wrong!